app/Models/User.ts

```diff
@@ -9,6 +18,12 @@ export default class User extends BaseModel {
   @column()
   public email: string
 
+  @column()
+  public username: string
+
+  @column()
+  public roleName: Roles
```

Comment:
roleId?

Reply:
ich dachte wir benutzen keine Nummer um die rolle abzubilden. sonst wird in der user tabelle nur die id stehen, ich habe jetzt den namen als key genommen

Reply:
Schau mal ins Wiki. Da haben wir das aufgeschrieben, wozu wir uns geeinigt haben ;)